adapter/handler/helper.go

```diff
@@ -7,7 +7,7 @@ import (
 	"google.golang.org/appengine/log"
 )
 
-func fail(ctx context.Context, w http.ResponseWriter, err error, code int) {
+func Fail(ctx context.Context, w http.ResponseWriter, err error, code int) {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function Fail should have comment or be unexported